datafusion/physical-expr/src/equivalence/ordering.rs

```diff
@@ -1065,4 +1066,63 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
+    fn test_ordering_satisfy_on_data() -> Result<()> {
```

Comment:
This looks to me like a fuzz test -- perhaps we could move it to https://github.com/apache/datafusion/tree/main/datafusion/core/tests/fuzz_cases/equivalence

ordering.rs perhaps

Reply:
This makes sense to me. I moved this test to under `.../fuzz_cases/equivalence/ordering.rs`